chainer/gradient_check.py

```diff
@@ -184,6 +186,8 @@ def check_backward(func, x_data, y_grad, params=(),
             :func:`assert_allclose`.
         rtol (float): Relative tolerance to be passed to
             :func:`assert_allclose`.
+        dtype (~numpy.dtype): `x_data` and `y_grad` are casted to this dtype
```

Comment:
Document says `y_grad` is also casted to specified `dtype`. But it seems to me there is no corresponding code.